cfme/tests/v2v/test_migration_throttling.py

```diff
@@ -0,0 +1,89 @@
+import fauxfactory
+import pytest
+from widgetastic.widget import NoSuchElementException
+
+from cfme import test_requirements
+from cfme.cloud.provider.openstack import OpenStackProvider
+from cfme.fixtures.provider import rhel69_template
+from cfme.fixtures.provider import rhel7_minimal
+from cfme.fixtures.provider import ubuntu16_template
+from cfme.infrastructure.provider.rhevm import RHEVMProvider
+from cfme.infrastructure.provider.virtualcenter import VMwareProvider
+from cfme.markers.env_markers.provider import ONE_PER_TYPE
+from cfme.markers.env_markers.provider import ONE_PER_VERSION
+from cfme.v2v.migration_settings import MigrationSettings as migration_settings
```

Comment:
So to distinguish between the class and the instance, I think you should keep the class name as `MigrationSettings` and call the instance `migration_settings`